mantidproject/mantid
# updated elemental analysis ref data

**Description of work.**
We are updating the elemental analysis peak data file. The user has provided it and we are uploading it to be the new default. The old one had errors. 


**To test:**
Open Elemental analysis
Right click Fe - a pop up will appear
Scroll down in the new window to the Gamma section
You will see the same transition multiple times (with additional spaces preceding it)

*There is no associated issue.*





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
